### DISCUSSION TITLE
Resource group names mixed between documentation and template page?

### DISCUSSION THREAD

**Discussion by baatch** [Azure/data-landing-zone#201] [Q&A] [answered]
The wording used for the resource groups shared integration and integration and processing are mixed between the documentation and the template example pages making it confusing and hard to follow.

Please see this documentation page: https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/data-management/architectures/data-landing-zone

![image](https://user-images.githubusercontent.com/16446781/138695357-2ae739a3-dc10-4f05-8469-81eac90a55da.png)

And this prereq page https://github.com/Azure/data-landing-zone/blob/main/docs/EnterpriseScaleAnalytics-Prerequisites.md for the template this picture is used:

![image](https://user-images.githubusercontent.com/16446781/138695766-e0146f71-e68d-465c-ad4c-86d231efcca6.png)

**Comment by marvinbuss** [comment]
Thanks for the feedback @baatch! This is something we should fix in docs. @abdale can you please look into it?

**Comment by abdale** [comment, marked as answer]
Thanks @baatch. We're working on a fix and will update the docs accordingly.
